packages/radix/src/components/Input.tsx

```diff
@@ -76,6 +76,27 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>((props, forw
                 lineHeight: theme.lineHeights[0], // Yields nice text selection height in Safari
                 paddingLeft: theme.space[2],
                 paddingRight: theme.space[2],
+
+                // A workaround for a tiny visual bug in Chrome
+                //
+                // When input is positioned ambiguously — imagine it's vertically centered on the screen
+                // so its position seems to be calculated with subpixel-level precision — there is a schism
+                // between how the actual input “box” is placed and how its text baseline is placed. In this
+                // case resizing the viewport height may introduce slight baseline misalignments and make
+                // text look not centered vertically within the input.
+                //
+                // This happens because actual input text is rendered by a shadow DOM element, which doesn't use
+                // the line height of the input. Instead, it uses the default "line-height: normal", which means 1.2.
+                //
+                // So:
+                // * The input height is 35px, the font size is 13px
+                // * The shadow DOM element height is Math.ceil(13px * 1.2) = 16px
+                // * 35px - 16px = 19px for the browser to allocate above and below the shadow DOM element
+                //
+                // Hence adding the 1px padding to resolve the mismatch between the odd input height (35px) and
+                // the even shadow DOM value element height (16px). This way baseline position stays consistent
+                // no matter how the input is placed.
+                paddingBottom: '1px',
               },
```

Comment:
I hope we support comments in Modulz 😅

Reply:
Well we also don’t expose padding do we? So 🤷‍♂️?

Reply:
@benoitgrelard looks like we eventually should expose it for one-line components, since that's the only way to fix baseline alignment issues with weird fonts and browser quirks like this one.

Reply:
- Can we expect people to do this though
- Isn’t seeing padding and height going to confuse people even more?
- It seems we’re trying to automate things in general in seemingly more complex primitives but perhaps this is the same here and we should try to do that too?

Reply:
Let's leave this as a quick fix just for Radix. It's a bad signalling of product quality to see a misaligned input on our login screen.

We'll need to figure out button and input alignment in Modulz soon, but let's not worry about that for now. Just wanna push a quick fix through here

Reply:
Agreed 👍 but I think we all need to keep in mind that our goal is to get rid of this hand coded radix eventually (and hopefully soon).

Reply:
Agree on all points @benoitgrelard 

> It seems we’re trying to automate things in general in seemingly more complex primitives but perhaps this is the same here and we should try to do that too?

Have been thinking about how this might work in Modulz since I started exploring this fix. I think it can be done with minor trade-offs, eventually.

There's at least two things that will force us come up with a solution when Radix is generated by Modulz — this input alignment issue and the smaller Button size, which relies on a specific line height right now, rather than `line-height: 1` that Modulz is doing.